docs/DeveloperGuide.adoc

```diff
@@ -12,9 +12,9 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-F13-4/main.git
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team CS2103T-F13-4`      Since: `Sept 2019`      Licence: `NUS`
 
 == Setting up
 
```

Comment:
Line 34 - the link "diagrams" does not work for your file. However, the original link works for the original addressBook3. Create a section "diagrams" in docs so that the link will work, with every diagram in the .puml form